frontend/src/pages/ShootItem.vue

```diff
@@ -326,11 +326,6 @@ limitations under the License.
             title: 'Cluster Autoscaler',
             description: 'Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernetes cluster.'
           },
-          {
-            name: 'heapster',
-            title: 'Heapster',
-            description: 'Heapster enables Container Cluster Monitoring and Performance Analysis.'
-          },
```

Comment:
I would keep this because old cluster can still have the heapster addon. Only new clusters should not use it anymore.

Reply:
good point